### PR TITLE
Feature#21/login design

### PR DESCRIPTION
**Pull request by jairolan**
# Description
What did you implemented/Why did you implemented this:
- I have added the stylesheet link in the application layout.
- I have designed the login. 
- I've coded the login view with mobile-first
 
 # Testing
 Help me how can I test or look at the changes
  
  ## Screenshots 
image
image

### CODE REVIEW

**Review by xFerchoVJ — APPROVED** [review]
LGTM!

File: local_sales/app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -1,26 +1,27 @@
-<h2>Log in</h2>
-
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+<div class="container max-w-md mx-auto xl:max-w-3xl h-full flex bg-white rounded-lg shadow overflow-hidden xl:mt-24 mt-16">
+  <div class="relative hidden xl:block xl:w-1/2 h-full">
+    <img class="absolute h-auto w-full object-cover" src="https://images.unsplash.com/photo-1606824722920-4c652a70f348?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1575&q=80"/>
```
Comment (juliocabrera820) [inline local_sales/app/views/devise/sessions/new.html.erb:3]: I think you can use a helper instead of img tag

**Review by juliocabrera820 — APPROVED** [review]
👍🏼

**Review by JoseRicardoJimenez — APPROVED** [review]
Nice!

**Pull request merged**